include/openenclave/internal/switchless.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef _OE_SWITCHLESS_H
+#define _OE_SWITCHLESS_H
+
+#include <openenclave/bits/defs.h>
+#include <openenclave/bits/types.h>
+#include <openenclave/internal/calls.h>
+
+typedef uint64_t oe_thread_t;
+
+typedef struct _switchless_settings
+{
+    size_t num_host_workers;
```

Comment:
nit: bad coding style to use abbreviations like "num".  Suggest "host_worker_count"

Reply:
The convention in OE seems to be using `num_`, such as `num_extensions`, `num_ocalls`.

Reply:
The Windows coding style conventions argue that it's a bad convention.  Specifically, using abbreviations in names is considered bad practice.

Reply:
I won't -1 a PR because of this, but I will continue to complain about instances of bad coding conventions.